.github/workflows/deploy.yml

```
@@ -0,0 +1,520 @@
+          python manage.py migrate --verbosity=2
+          
+          # Verify database integrity
+          python manage.py check --deploy
+          
+          # Test database connectivity
+          python -c "
+          import django
+          django.setup()
+          from django.db import connection
+          from django.contrib.auth.models import User
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)